spec/models/user_spec.rb

```diff
@@ -34,43 +34,11 @@
 
   describe "validations" do
     it { is_expected.to validate_presence_of(:full_name).with_message("Enter a full name") }
-    it { is_expected.to validate_presence_of(:email).with_message("Enter an email address") }
+    it { is_expected.to validate_presence_of(:email).on(:npq_separation).with_message("Enter an email address") }
```

Comment:
we removed the on separation logic, worth updating with main

Reply:
removed